addons/a11y/src/components/ColorBlindness.tsx

```diff
@@ -8,6 +8,16 @@ import { Icons, IconButton, WithTooltip, TooltipLinkList } from '@storybook/comp
 
 const getIframe = memoize(1)(() => document.getElementById('storybook-preview-iframe'));
 
+const getFilter = (filter: string | null) => {
+  if (filter === null) {
```

Comment:
is `!filter` better?

Reply:
It could, but I wanted to be "strict" and consistent, since the reset is `this.setFilter(null);` and there's a similar check `if (filter !== null) {`